crates/api_common/src/utils.rs

```diff
@@ -756,15 +775,22 @@ pub async fn purge_user_account(
   context: &LemmyContext,
 ) -> Result<(), LemmyError> {
   let pool = &mut context.pool();
-  // Delete their images
+
   let person = Person::read(pool, person_id).await?;
+
+  // Delete their local images, if they're a local user
+  purge_local_user_images(person_id, context).await.ok();
```

Comment:
It would be good if you can move all the functions related to purging/removing images and posts into a separate file, then we can see it all in one place and simplify it. Some of it definitely seems duplicated. Were also missing api tests for purging functionality.

Reply:
No, it extracted delete_images_for_local_user into a common function, and used it in one more place, L782 in this file. It was missing before.